__tests__/index.test.js

```diff
@@ -42,4 +35,121 @@ describe('GET /api', () => {
             expect(endpoints).toEqual(jsonEndpoints);
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('GET: 200 returns the article id that is input into the endpoint', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article;
+            expect(article.article_id).toEqual(1);
+        })
+    })
+    test('GET: 200 returns the article id that is input into the endpoint', () => {
+        return request(app)
+        .get('/api/articles/5')
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article;
+            expect(article.article_id).toEqual(5);
+        })
+    })
+    test('GET: 200 responds by sending the article object back to the user according to the id on the endpoint', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article;
+            expect(article).toMatchObject({
+                article_id: 1,
```

Comment:
If you're checking the id here, you probably don't need the separate test for article_id